src/cli/global/update.rs

```diff
@@ -66,11 +65,8 @@ pub async fn execute(args: Args) -> miette::Result<()> {
 
         // Remove outdated binaries
         state_changes |= project.prune_exposed(env_name).await?;
-        eprintln!(
-            "{}Updated environment: {}.",
-            console::style(console::Emoji("✔ ", "")).green(),
-            env_name.fancy_display()
-        );
+
+        state_changes.report();
```

Comment:
The report already happens on line 99. I think this should be reverted

Reply:
that report is applied on newer StateChanges object ( it is init on line 82 )